register-system.yaml

```diff
@@ -65,12 +65,16 @@
       shell: >-
         ipmitool lan print | awk -F' *: *' '$1 == "IP Address" {print $2}'
       register: ipmi_addr
+      when: ipmi_ip is not defined
+
+    - name: "Set IPMI IP"
+      set_fact:
+        ipmi_ip: "{{ ipmi_addr.stdout }}"
```

Comment:
Using `ipmi_addr` vs `ipmi_ip` as variable names seems like a recipe for confusion (they both mean the same thing to me). I would go for maybe something like `ipmi_ip_cmd` instead of `ip_addr`.

Reply:
good idea, i'll change that.